compiler/src/main/scala/dx/compiler/WorkflowCompiler.scala

```diff
@@ -167,6 +177,168 @@ case class WorkflowCompiler(extras: Option[Extras],
     (commonMeta ++ workflowMeta, commonDetails)
   }
 
+  private def listMd(s: String): String = {
+    "\n* " + s
+  }
+
+  private def listMd2(s: String): String = {
+    "\n    * `" + s + "`"
+  }
+
+  // Summarize reportable dependencies from details of a workflow
+  private def reportableDependenciesFromWorkflow(
+      workflow: DxExecutable
+  ): Option[String] = {
+    val description = workflow.describe(Set(Field.Name, Field.Details, Field.Hidden)) match {
+      case w: DxWorkflowDescribe => w
+      case _                     => throw new Exception(s"Expected ${workflow.id} to be a workflow.")
+    }
+
+    // Do not generate description section if hidden
+    if (description.hidden.isDefined && description.hidden.get) {
```

Comment:
Replace with `description.exists(_)`

Reply:
Also this does nothing. You either need to `return` here or wrap the rest of the code in an `else` block.

Reply:
Like this? 

```
    if (description.hidden.exists(h => h)) {
      return None
    }
```

Reply:
yes, although `exists(_)` is equivalent